### ISSUE TITLE
what can the depth esitimation picture do?

### ISSUE THREAD

**Issue by Asherchi** [iro-cp/FCRN-DepthPrediction#94]
I am confused about I did get the same depth estimation pictures as yours,but I dont know how can i use it.it can get the truth distance between object and camera?if it can get the distance and can you show me the detail?

**Comment by Saoussenl** [comment]
Yes, you can use the depth value (in meters) by accessing depth_map[:, y, x, 0][0] for the pixel of coordinates (x, y). I have looked for this information on their paper but there seems no explanation so I just read the code in predict.py.

**Comment by eugeneYz** [comment]
Do u have the download url of pretrained model ?

**Comment by Asherchi** [comment]
> Do u have the download url of pretrained model ?

sorry, it is too long ago, i don't remenber the detail.

**Comment by eugeneYz** [comment]
> Yes, you can use the depth value (in meters) by accessing depth_map[:, y, x, 0][0] for the pixel of coordinates (x, y). I have looked for this information on their paper but there seems no explanation so I just read the code in predict.py.

Hi, I changed the input size as width 480, height 640, intput the 640*480 picture, but the type of pred is (1,240,320,1), so how can I get the depth value of pixel of coordinates (x, y).
